presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/PostgreSqlQueryRunner.java

```diff
@@ -49,11 +49,23 @@ public static DistributedQueryRunner createPostgreSqlQueryRunner(
             Map<String, String> connectorProperties,
             Iterable<TpchTable<?>> tables)
             throws Exception
+    {
+        return createPostgreSqlQueryRunner(server, extraProperties, connectorProperties, tables, 3);
+    }
+
+    public static DistributedQueryRunner createPostgreSqlQueryRunner(
```

Comment:
I would rather drop this commit.
The added burden of an overload is not justified for arguably small benefit this brings.